package.json

```diff
@@ -68,5 +68,11 @@
     "tape": "^4.6.3",
     "tape-spawn": "^1.4.2",
     "temporary-directory": "^1.0.2"
+  },
+  "pkg": {
+    "assets": [
+      "node_modules/utp-native/prebuilds/**",
```

Comment:
Found an explanation here https://github.com/sindresorhus/globby#globbing-patterns , apparently ** is inclusive of / and * is not https://github.com/sindresorhus/multimatch/blob/master/test/test.js#L38